src/app/state/wallet/saga.ts

```diff
@@ -33,6 +35,8 @@ export function* rootWalletSaga() {
 
   // Reload wallet balances if network changes
   yield* takeEvery(networkActions.networkSelected, refreshOnNetworkChange)
+
+  yield* takeEvery(walletActions.verifyAddressOnLedger, verifyAddressOnLedger)
```

Comment:
> when click multiple times at verify button it throws error, but probably users won't do this

oops. Switching this to `takeLeading` to fix it. Sometimes parallel commands to ledger cause it to get stuck, so worth preventing it